application/src/main/resources/thingsboard.yml

```diff
@@ -100,6 +100,8 @@ server:
     rate_limits:
       # Limit that prohibits resetting the password for the user too often. The value of the rate limit. By default, no more than 5 requests per hour
       reset_password_per_user: "${RESET_PASSWORD_PER_USER_RATE_LIMIT_CONFIGURATION:5:3600}"
+    rule_engine:
```

Comment:
Hi @devaskim.
Thank you for your contribution!

Please add description for the property.